style.css

```diff
@@ -0,0 +1,226 @@
+
+body {
+    margin: 0;
+ }
+
+/*HEADER*/
+
+.main-header {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: white;
+    border: 2px solid black;
+}
+
+.nav-menu {
+    display: none;
+}
+
+.logo-ticko {
+    display: grid;
+    grid-column-start: 1;
+    align-self: center;
+}
+
+.icon-menu {
+    display: grid;
+    grid-column-start: 2;
+    place-self: center;
+}
+
+/*NAVBAR*/
+
+    a {
+    font-size: 24px;
+    text-decoration: none;
+    margin: 10px;
+    }
+
+    a:link, a:visited {
+    text-decoration: none;
+    }
+    
+    a:hover, a:active {
+    color: rgb(110, 8, 8);
+    font-size: 110%;
+    }
+
+    .button {
+        background-color:#6a65fa;
+        border: none;
+        color: white;
+        padding: 10px 20px;
+        text-align: center;
+        text-decoration: none;
+        display: inline;
+        font-size: 16px;
+        margin: 4px 2px;
+        cursor: pointer;
+        border-radius: 1rem;
+        text-align: end;
+    }
+
+    .picture-putin {
+        width: 300px;
+        margin: 0;
+    }
+
+    .grid-main-news {
+        display: grid;
+        grid-template-columns: 1fr;
+        background-color: #dbdaf8;
+        border-top: 1px solid black;
+        border-bottom: 1px solid black;
+    }
+
+/*SMÅ NYHETERNA*/
+
+    .grid-item1 {
+        background-color: white;
+        padding: 10px;
+        border-bottom: 1px solid black;
+        border-top: 1px solid black;
+    }
+    
+    .grid-item2 {
+        background-color: #dbdaf8;
+        padding: 10px;
+        border-bottom: 1px solid black;
+        border-top: 1px solid black;
+    }
+    
+    .picture-virus {
+        width: 200px;
+        height: auto;
+    }
+    
+    
+    .grid-item3 {
+        background-color: white;
+        padding: 10px;
+        border-bottom: 1px solid black;
+        border-top: 1px solid black;
```

Comment:
to make code more simple you could also se the borders to the grid-items' parent (=grid-small-news) instead of to the every grid item separately, just an idea :)